uint8 ERROR = 0
uint8 WARNING = 1
uint8 INFO = 2

uint8 type
string text